src/content/glossary.md

```diff
@@ -79,6 +79,7 @@ This index lists common terms used throughout the webpack ecosystem.
 ## S
 
 - [__Scaffolding__](/guides/scaffolding/): This feature allows creating a webpack configuration by using customizable third-party initialization packages.
+- [__Shimming__](/guides/shimming/): It is likely that some third party libraries that you're depending on may expect global dependencies (e.g. $ for jQuery). They might also create globals which need to be exported. These "broken modules" are one instance where shimming comes into play. Another instance where shimming can be useful is when you want to polyfill browser functionality to support more users.
```

Comment:
This is not a definition of shimming, this is a copy of text from shimming guide